features/admin.administrators.v1/package.json

```diff
@@ -30,7 +30,7 @@
         "@mui/material": "^5.13.0",
         "@mui/system": "^5.12.3",
         "@mui/utils": "^5.12.3",
-        "@oxygen-ui/react": "^1.13.2",
+        "@oxygen-ui/react": "^1.13.6",
```

Comment:
Don't we need to bump this in other modules?

Reply:
Those are already bumped.

Reply:
fixed with https://github.com/wso2/identity-apps/pull/6700/commits/2f9ead44743ee603856ee9d29924c6595497540a